### PR TITLE
Fix GCE live tests

### PR DESCRIPTION
**Pull request by nacx**
Fixes all Google Compute Engine Live tests: :muscle:

```bash
Tests run: 179, Failures: 0, Errors: 0, Skipped: 8
Total time: 01:04 h (Wall Clock)
```

### CODE REVIEW

**Comment by andreaturli** [comment]
is the builder failure related?

rebuild please

**Comment by nacx** [comment]
Yes, same Jenkins plugin bug :(

**Comment by nacx** [comment]
I've upgraded the plugins and it should work now: rebuild please!

File: providers/google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/functions/ResetWindowsPassword.java [hunk 1]
```diff
@@ -135,31 +146,64 @@ public String apply(Map<String, ?> params) {
...
+
+               return retrievedPassword.isPresent();
+            }
+            // Notice that timeoutDuration should be less than EXPIRE_DURATION
+         }, 10 * 60, 30, TimeUnit.SECONDS).apply(instance.get());
+
+         if (passwordRetrieved) {
+            return decryptPassword(encryptedPassword.get(), keys);
+         } else {
+            throw new IllegalStateException("Did not find the encrypted password in the serial port output");
```
Comment (andreaturli) [inline providers/google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/functions/ResetWindowsPassword.java]: should we also logged here?

**Comment by nacx** [comment]
merged to [master](http://git-wip-us.apache.org/repos/asf/jclouds/commit/715994b1) and [2.0.x](http://git-wip-us.apache.org/repos/asf/jclouds/commit/b0fa8abb).

File: providers/google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/functions/ResetWindowsPassword.java (hunk 1 above)
Comment (nacx) [reply providers/google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/functions/ResetWindowsPassword.java]: Why not. I'll add in another commit!

**Pull request closed without merging**